l10n_it_vat_statement_communication/models/config.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+
+from odoo import models, fields
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    vsc_supply_code = fields.Char(
+        'Vat statement communication supply code',
+        help="ES: IVP18")
```

Comment:
Aggiungerei il default a 'IVP18', visto che da specifiche ora il valore è fisso.

Inoltre il campo `vsc_supply_code` va poi effettivamente usato nella generazione dell'XML, in `x1_1_1_CodiceFornitura.text`

Reply:
ok procedo, ma quindi anche la PR 11 sarebbe da sistemare?

Reply:
Sì è un errore